packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterDialog.tsx

```diff
@@ -104,14 +111,36 @@ function PresentationInstanceFilterDialogContent(props: PresedntationInstanceFil
   const { onApply, initialFilter, descriptor, imodel, ruleGroupDepthLimit, filterResultCountRenderer, onClose } = props;
   const [initialPropertyFilter] = useState(() => (initialFilter ? convertPresentationFilterToPropertyFilter(descriptor, initialFilter.filter) : undefined));
 
-  const { rootGroup, actions, validate } = usePropertyFilterBuilder({
+  const numericInputValidator = (item: PropertyFilterBuilderRule) => {
+    if (
+      item.value &&
+      item.value.valueFormat === PropertyValueFormat.Primitive &&
+      item.property &&
+      (item.property.typename === StandardTypeNames.Number ||
+        item.property.typename === StandardTypeNames.Int ||
+        item.property.typename === StandardTypeNames.Float ||
+        item.property.typename === StandardTypeNames.Double) &&
+      item.value.value === undefined &&
+      item.value.displayValue !== ""
```

Comment:
Would it be possible to reduce conditions in this `if`? Maybe move some to separate functions (like checking if property is numeric)

Reply:
moved checking if property is numeric to other function